src/pandas_profiling/model/statistic.py

```diff
@@ -0,0 +1,591 @@
+"""Compute statistical description of datasets."""
```

Comment:
summary.py might be a better name for the file. Items in the description set are often statistical, but they do not have to be.